actor/src/main/java/org/apache/pekko/japi/pf/FSMStateFunctionBuilder.java

```diff
@@ -54,7 +54,7 @@ private FSMStateFunctionBuilder<S, D> erasedEvent(
         FSM.Event.class,
         new Predicate<FSM.Event>() {
           @Override
-          public boolean test(FSM.Event e) {
+          public boolean test(FSM.Event e) throws Exception {
```

Comment:
I will rewrite these kinds to lambda once https://github.com/apache/pekko/pull/1986 gets merged